From: Rene van Paassen <Rene.vanPaassen@gmail.com>
Date: 2014-03-11 16:34:05 +0100
Subject: failure to run new cmake 3.0 on ogre
References: 
Upstream: tbd, cmake is a development release, might also change

new cmake (3.0) no longer accepts [ ] in the COMMAND ARGS string

Don't know where to submit, since it might become cmake policy, then -> ogre, 
or not, then -> cmake


Index: sinbad-ogre-dd30349ea667/Samples/Browser/CMakeLists.txt
===================================================================
--- sinbad-ogre-dd30349ea667.orig/Samples/Browser/CMakeLists.txt
+++ sinbad-ogre-dd30349ea667/Samples/Browser/CMakeLists.txt
@@ -339,7 +339,7 @@ if (APPLE)
 
     foreach(FWK ${FRAMEWORKS})
 	add_custom_command(TARGET SampleBrowser POST_BUILD
-        COMMAND ARGS if [ -d ${OGRE_BINARY_DIR}/lib/${OGRE_OSX_BUILD_CONFIGURATION}/${FWK}.framework ]\; then ln -s -f ${OGRE_BINARY_DIR}/lib/${OGRE_OSX_BUILD_CONFIGURATION}/${FWK}.framework ${OGRE_SAMPLE_CONTENTS_PATH}/Frameworks/${FWK}.framework\; fi
+        COMMAND ARGS if test -d ${OGRE_BINARY_DIR}/lib/${OGRE_OSX_BUILD_CONFIGURATION}/${FWK}.framework; then ln -s -f ${OGRE_BINARY_DIR}/lib/${OGRE_OSX_BUILD_CONFIGURATION}/${FWK}.framework ${OGRE_SAMPLE_CONTENTS_PATH}/Frameworks/${FWK}.framework\; fi
 		)
     endforeach(FWK)
 
